lessons/pydata/ml_intro/index.ipynb

```diff
@@ -0,0 +1,492 @@
+{
```

Comment:
<p>Já bych byl moc rád, kdybychom používali objektový matplotlib a nikoli globálně-stavový přístup. Tady třeba</p><p><code></code><code></p><p>plt, ax = plt.subplots()</p><p>ax.scatter(...)</p><p>ax.set_xlabel(...)</p><p>...</p><p></code><code></code></p><p>Tohle se totiž daleko lépe čte, škáluje na složitější věci a vede méně zašmodrchaným chybám.</p><p> </p><p>Asi to není ale to nejdůležitější :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
OK, opravím.